### DISCUSSION TITLE
Should/can ignoreDeps be case insensitive?

### DISCUSSION THREAD

**Discussion by c0state** [renovatebot/renovate#10126] [Request Help]
It looks like `ignoreDeps` is currently "exact match" only, which implies that it is case sensitive as well. However, the dependencies in some (eg: Python/`pyproject.toml`), or maybe most ecosystems are case insensitive. 

Does it make sense to make it insensitive for applicable languages? Or is there a plan to switch to regex? If so that would probably supersede this discussion.

**Comment by rarkins** [comment]
I don't think we've thought about it before. It will be matched against whatever casing you use in your repo (e.g. `requirements.txt` and not what's used on PyPI). npm is *now* case sensitive but does historically have some "duplicate" packages with different casing. I can't think of a realistic case where someone's using both and would not want case-insensitive matching.
